docs/contribute/environment.md

```diff
@@ -93,7 +93,7 @@ There are two primary ways to setup a development environment.
 
     1. Install your local copy of `ibis` into the Conda environment. In the root of the project run:
 
-            pip install -e .
```

Comment:
The `-e` here is referring to the `--editable` install, not easy install 😅

Reply:
Oh! When I ran the command locally it gave me an error, that said easy installed has been depreciated. The command worked when I removed the -e flag. Not sure if it was just an error for me.

Reply:
I'll close this PR if its just something on my end, though I'd be curious to know why that happened 😅

Also, just for in the future to know @cpcloud is there a way to edit text once you've run `cz commit`? CI looks like its failing because the body text I added is too long. I'm not sure how I would fix this if it ever happened again without creating a new PR.

Reply:
> is there a way to edit text once you've run cz commit?

Yep! `cz commit` ultimately just runs `git commit`, so the usual git things still work.

In this case you can run `git commit --amend` which will open up an editor with the text of the commit message from your current `HEAD` which you can edit to modify that commit's message.

Reply:
> I'd be curious to know why that happened

This I'm not too sure about. Can you show the output from running the command?

Reply:
This is the main part of the error, I just assumed it was an easy install error but taking a look at it now it looks like it might be something else. 

```
Running setup.py develop for ibis-framework
    error: subprocess-exited-with-error
    
    × python setup.py develop did not run successfully.
    │ exit code: 1
    ╰─> [7 lines of output]
        running develop
        /opt/homebrew/Cellar/python@3.9/3.9.9/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/setuptools/command/easy_install.py:156: EasyInstallDeprecationWarning: easy_install command is deprecated. Use build and pip and other standards-based tools.
          warnings.warn(
        /opt/homebrew/Cellar/python@3.9/3.9.9/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
          warnings.warn(
        running egg_info
        error: package directory 'ibis/backends/base/file' does not exist
        [end of output]
    
    note: This error originates from a subprocess, and is likely not a problem with pip.
  WARNING: No metadata found in /opt/homebrew/lib/python3.9/site-packages
  Rolling back uninstall of ibis-framework
  Moving to /opt/homebrew/lib/python3.9/site-packages/ibis/
   from /opt/homebrew/lib/python3.9/site-packages/~bis
  Moving to /opt/homebrew/lib/python3.9/site-packages/ibis_framework-2.1.1.dist-info/
   from /opt/homebrew/lib/python3.9/site-packages/~bis_framework-2.1.1.dist-info
error: subprocess-exited-with-error

× python setup.py develop did not run successfully.
│ exit code: 1
╰─> [7 lines of output]
    running develop
    /opt/homebrew/Cellar/python@3.9/3.9.9/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/setuptools/command/easy_install.py:156: EasyInstallDeprecationWarning: easy_install command is deprecated. Use build and pip and other standards-based tools.
      warnings.warn(
    /opt/homebrew/Cellar/python@3.9/3.9.9/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
      warnings.warn(
    running egg_info
    error: package directory 'ibis/backends/base/file' does not exist
    [end of output]

note: This error originates from a subprocess, and is likely not a problem with pip.
```

Reply:
> > is there a way to edit text once you've run cz commit?
> 
> Yep! `cz commit` ultimately just runs `git commit`, so the usual git things still work.
> 
> In this case you can run `git commit --amend` which will open up an editor with the text of the commit message from your current `HEAD` which you can edit to modify that commit's message.

Awesome! Thank you!!!! I have a few other things I'd like to add to the docs in this section, so I'll test all this out in a new PR 😄

Reply:
The main error here is:

```
error: package directory 'ibis/backends/base/file' does not exist
```

which is unrelated to easy install (which appears to be just a warning, though seeing the initial command might help explain this).

The error message suggests that your ibis checkout is a bit old, since we removed the file-based backends in #3414, #3420, and #3422 so the `ibis/backends/base/file` directory no longer exists.

Reply:
That makes sense! I missed that. Thanks, I'll update my main branch!

Reply:
No worries, happy to work through it!